### ISSUE TITLE
Ambiguous use of 'USER_ID

### ISSUE THREAD

**Issue by UtkuDalmaz** [QuickBlox/quickblox-ios-sdk#503]
I am trying to Block user and using this code

```
let item = QBPrivacyItem(type: USER_ID, valueForType: 1022685, action: DENY)
let list  = QBPrivacyList(name: "public", items: [item])

        QBChat.instance().setPrivacyList(list)
        QBChat.instance().setDefaultPrivacyListWithName("public")
        QBChat.instance().setActivePrivacyListWithName("public")
```
 
However i get `Ambiguous use of 'USER_ID'` error 
How can i fix it ?

**Comment by Raikerian** [comment]
Hi,

I have tried to reproduce your issue and was not successful to do it. It is ok for me. Please check that you do not have any other type of definition for `USER_ID`

**Comment by UtkuDalmaz** [comment]
I tried everything but cannot fix this issue. 

I get this error for both type (USER_ID) and action (DENY)

Any suggestion or help ?

**Comment by Raikerian** [comment]
Can you send us a test project with this issue been reproducible?

**Comment by UtkuDalmaz** [comment]
I sent the project to your email address

**Comment by Raikerian** [comment]
For some reason your project does not want to see enums that we have in QBPrivacyItem.h header file. We will improve it by changing them with NS_ENUM's in next release.

Right now there is a workaround for you.
Use raw value instead of actual enum names.

Examples:
`QBPrivacyItemType(rawValue: 1)` instead of `USER_ID`
`QBPrivacyItemAction(rawValue: 1)` instead of `DENY`

raw values of QBPrivacyItemType:
```objectivec
USER_ID = 1
GROUP_USER_ID = 2
GROUP = 3
SUBSCRIPTION = 4
```

raw values of QBPrivacyItemAction:
```objectivec
ALLOW = 0
DENY = 1
```

**Comment by UtkuDalmaz** [comment]
Thank you

**Comment by UtkuDalmaz** [comment]
I have one more problem regarding this issue

```
 func chatDidReceivePrivacyList(privacyList: QBPrivacyList) {
        self.blockbtn.setTitle(NSLocalizedString("Block", comment: ""), forState: .Normal)
        for item in privacyList.items {
            let blockedid = item.valueForType
            let status = item.action
            if (self.userQBID == UInt(blockedid) && status == "deny") {
                 self.isblocked = true
                self.blockbtn.setTitle(NSLocalizedString("Unblock", comment: ""), forState: .Normal)
            }
        }
    }
```
I also cannot use item.action and i get Ambiguous use of 'action' 
is there any workaround for this ?

**Comment by Raikerian** [comment]
Seems like a bug too, we will take a look.

Here is a possible working solution for you for now (cast item as QBPrivacyItem):

```swift
privacyList.items.forEach { (item) in
            
            let privacyItem: QBPrivacyItem = item as! QBPrivacyItem
            
            let blockedid = privacyItem.valueForType
            let status = privacyItem.action
            if (self.userQBID == UInt(blockedid) && status == "deny") {
                self.isblocked = true
                self.blockbtn.setTitle(NSLocalizedString("Unblock", comment: ""), forState: .Normal)
            }
}
```

**Comment by UtkuDalmaz** [comment]
it should be 

` if (self.userQBID == UInt(blockedid) && status == QBPrivacyItemAction(rawValue: 1)) {`

Thank you again

**Comment by Raikerian** [comment]
This issue was fixed in 2.7.4.

If you are still having any other issues, feel free to create a separate issue.
